How to build and run the application

ctolon22 opened the pull request.

RecuencoJones commented on description.md:
```diff
@@ -117,4 +124,5 @@ As we can see our project follows Mavens directory layout. There are some direct
-- [Introduction to the Standard Directory Layout with Maven](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html)
\ No newline at end of file
+- [Introduction to the Standard Directory Layout with Maven](https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html)
+>>>>>>> upstream/master
```
remove this line

RecuencoJones commented on description.md:
```diff
@@ -109,6 +109,13 @@ In our case, our project has the next structure:
 
+##The next instructions will help you to build and run the application:
+
+The first step is to download the project or clone with git (`git clone https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race`). Now, you can see the code and this information. This code is built with gradle. Gradle is a build automation system which uses configuration files like build.gradle which is a build configuration script. To build the code you have to use gradle: `gradle build` on ...
```
this is duplicated, there is [another section](https://github.com/ctolon22/lab1-git-race/blob/master/description.md#how-to-run-the-application-on-the-local-machine) explaining this, please try to improve it

ctolon22 commented:
I have changed it. I have added a new section "How to build the code in the local machine" and i have omitted information related to "run the application" because is in the section "How to run the application  in the local machine"

SAReyes commented:
I find this info kind of meaningless, I believe most of us can get a quick look at [gradle's userguide](https://docs.gradle.org/current/userguide/userguide) to find this. I'll let other reviewers decide on this. 

I do get the main purpose of this project is to learn about a project lifecycle, but it bugs me when someone adds documentation for third party components without even referencing it out.

RecuencoJones commented on description.md:
```diff
@@ -47,22 +47,26 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
 # HOW TO RUN THE APPLICATION ON THE LOCAL MACHINE
 
-It's easy to run the application on the local machine. You just have to follow the wiki of the main repository to configure the tools needed. After configuiring Gradle you can try to compile using the command `gradle check` in the terminal. Once this is done, you can start the application using the command `gradle bootrun`. If everything goes fine, the application will have been compiled and star ...
+To build the project is required to download it or clone with git (`git clone https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race`). Now, you can see the code and this information. This code is built with gradle. Gradle is a build automation system which uses configuration files like build.gradle which is a build configuration script. You just have to follow the wiki of the main reposito ...
```
- `To build the project, first download it or clone...`
- `...which uses build configuration scripts like build.gradle.`

RecuencoJones commented on description.md:
```diff
@@ -47,22 +47,26 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
-It's easy to run the application on the local machine. You just have to follow the wiki of the main repository to configure the tools needed. After configuiring Gradle you can try to compile using the command `gradle check` in the terminal. Once this is done, you can start the application using the command `gradle bootrun`. If everything goes fine, the application will have been compiled and star ...
+To build the project is required to download it or clone with git (`git clone https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race`). Now, you can see the code and this information. This code is built with gradle. Gradle is a build automation system which uses configuration files like build.gradle which is a build configuration script. You just have to follow the wiki of the main reposito ...
+
+After configuring Gradle you can try to compile using the command `gradle check` in the terminal, this will compiles and tests your code. Other command you can use to build the code is `gradle build` on your terminal. Gradle will compile and check your code and create a JAR file containing your main classes and resources. 
```
`This will compile and test`*

RecuencoJones commented on description.md:
```diff
@@ -47,22 +47,26 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
+
+After configuring Gradle you can try to compile using the command `gradle check` in the terminal, this will compiles and tests your code. Other command you can use to build the code is `gradle build` on your terminal. Gradle will compile and check your code and create a JAR file containing your main classes and resources. 
+
+Other possibilities related to `gradle build` are `gradle assemble` that compiles and jars your code, but does not run the unit tests. After a few seconds, gradle respond you with "BUILD SUCCESSFUL" that means the build has completed and you are able to run the application. 
```
- You mention other possibilities but just mention one (assemble), either replace `are` for `like` or remove the plural or keep it single.
- `gradle will respond with`*

RecuencoJones commented:
Looks good to me

![LGTM](https://lgtm.in/p/u5lGFSl1q)

fjlopez commented:
![LGTM](https://lgtm.in/p/CXm2Vb2Rp)
